### PR TITLE
add params to get requests

### PR DESCRIPTION
**Pull request by dottodot**
### Summary

(If you have not already please refer to the contributing guideline as [described
here](https://github.com/feathersjs/feathers/blob/master/.github/contributing.md#pull-requests))

- [X] Tell us about the problem your pull request is solving.
- [ ] Are there any open issues that are related to this?
- [ ] Is this PR dependent on PRs in other repos?

### Other Information

Stripe has the option to expand fields i.e on a customer
```
expand: ['invoice_settings.default_payment_method']
```
but currently the get requests don't accept params so this isn't possible. This PR will resolve this.

### CODE REVIEW

File: lib/services/subscription.js [hunk 1]
```diff
@@ -1,31 +1,34 @@
-const errorHandler = require('../error-handler');
-const normalizeQuery = require('../normalize-query');
-const Base = require('./base');
+const errorHandler = require("../error-handler");
+const normalizeQuery = require("../normalize-query");
+const Base = require("./base");
```
Comment (daffl) [inline lib/services/subscription.js]: Formatting should say consistent with the linter set up in the repository (it should fix automatically if you run `npm test` locally and the commit).

Comment (dottodot) [reply lib/services/subscription.js]: Sorry about that, should be ok now.

**Comment by Datebayo350** [comment]
hello @dottodot this PR is still open so your adds are not implemented for the moment, am i right ?
I passed a couple of hours trying to make the expand property work, until i found this PR .... Any other solutions have been found ?

Thanks

**Comment by DaddyWarbucks** [comment]
I have made some updates in PR: https://github.com/feathersjs-ecosystem/feathers-stripe/pull/107

The updates made in that PR may likely clash with these. But, I am happy to update this PR (or open another) to accomplish this. It's definitely a worthwhile addition to the library.

**Pull request closed without merging**